### PR TITLE
Created toys show page

### PR DESCRIPTION
**Pull request by 7chris71000**
Fixes issue #23

### CODE REVIEW

File: app/controllers/toys_controller.rb [hunk 1]
```diff
@@ -1,32 +1,41 @@
...
 
+	def show
+		@title = "Edit Toy"
+		@toy = Toy.find(params[:id])
+
+		# random number list to populate 5 'related' toys under product
+		# array is populated with 5 unique random numbers used to find id from DB
+		@randomNumArray = [] 
+		i = 0 # counter to determine when 5 numbers have been added to array
+		largestId = Toy.last.id
```
Comment (caseyli) [inline app/controllers/toys_controller.rb]: @7chris71000 same here:

```largest_id = Toy.last.id```

File: app/controllers/toys_controller.rb [hunk 2]
```diff
@@ -1,32 +1,41 @@
...
+		while i < 5 do  
+			randNum = rand(1..largestId)
```
Comment (caseyli) [inline app/controllers/toys_controller.rb]: @7chris71000 same here:

```random_number = rand(1...largest_id)```

File: app/views/toys/index.html.erb [hunk 3]
```diff
@@ -17,7 +17,7 @@
 	          					
 	          					<div class="book-cover-container">
 	          						<%= toy.image_url.present? ? image_tag(toy.image_url, width: '160') : image_tag("no_image_found.jpg", height: '238', width: '160') %>
-	          						<%= link_to "View Details", "#", class: "view-details" %>
+	          						<%= link_to "View Details", "toys/#{toy.id}", class: "view-details" %>
```
Comment (caseyli) [inline app/views/toys/index.html.erb]: @7chris71000 it's a good habit to link your routes relative to the root, so in this case, you're just missing a forward slash:

```<%= link_to "View Details", "/toys/#{toy.id}", ... %>```

**Comment by 7chris71000** [comment]
Changes have been applied and pushed.

**Pull request merged**